src/designSystem/components/Tooltip/Tooltip.tsx

```diff
@@ -161,22 +175,47 @@ export const Tooltip = ({
   orientation,
 }: TooltipProps) => {
   const controlRef = useRef<HTMLDivElement | null>(null);
+  const timeoutRef = useRef<any>(null);
   const [controlRect, setControlRect] = useState<DOMRect | null>(null);
   const [isOpen, setIsOpen] = useState(false);
 
   const internalOnMouseEnter = useCallback(() => {
     if (controlRef.current) {
       const rect = controlRef.current.getBoundingClientRect();
-      setControlRect(rect);
-      setIsOpen(true);
+
+      /**
+       * This forces a commit to hover by the user instead of
+       * firing the tooltip as they idly move their mouse
+       * across the screen. The delay is short enough to not
+       * be too noticeable.
+       */
+      timeoutRef.current = setTimeout(() => {
+        setControlRect(rect);
+        setIsOpen(true);
+      }, 300);
```

Comment:
I fiddled with this until it felt right. At 300ms it prevents accidental tooltip opening, but its hardly noticeable unless you're looking for it.